test_util/marathon.py

```diff
@@ -128,7 +128,12 @@ def deploy_test_app_and_check(self, app, test_uuid):
                 msg += "Detailed explanation of the problem: {2}"
                 raise Exception(msg.format(r.status_code, r.reason, r.text))
 
-            assert r.json() == {'username': marathon_user}
+            json_uid = r.json()['uid']
+            if marathon_user == 'root':
+                assert json_uid == 0
+            else:
+                assert json_uid != 0
```

Comment:
Let's add some messages to these asserts, something like
`assert json_uid == 0, "App running as root should have uid 0."`
`assert json_uid != 0, "App running as <marathon_user> should not have uid 0."`